src/v2/controllers/FormController.js

```diff
@@ -147,7 +149,19 @@ export default Controller.extend({
     }
 
     this.toggleFormButtonState(true);
+
+    // NOTE: okta-idx-js does not know whether remediations
+    // are required to redirect or perform a GET request.
+    // As a result, we handle this accordingly by checking the formName
+    const fullPageRedirectForms = [ FORMS.REDIRECT_IDP ];
+    if (fullPageRedirectForms.includes(formName)) {
+      const currentViewState = this.options.appState.getCurrentViewState();
+      Util.redirectWithFormGet(currentViewState.href);
+      return;
```

Comment:
also i don't like the idea of breaking the remediation process here, specially if the user actions ended up successful

Reply:
Can you elaborate a bit more on this concern?

From what I can see, we're not breaking the remediation process here, we're just handling it via navigation instead of passing the `formName` into `idx.proceed`.

Reply:
exactly that's my point.. we are circumventing `idx.proceed` and i feel it should not do that.

changing idx.js for this use case right now could be overkill and i don't think this PR should be blocked because of that. We can create a new jira for Idx.js change.

Can we handle this use case after passing it to idx.proceed, in `handleIdxResponseSuccess` ? Also should we handle failure in redirection ?

Reply:
Yep - agree with your concerns and believe `idx-js` is a good place to handle this (more info called out [here](https://github.com/okta/okta-signin-widget/pull/1638#discussion_r572211893)). We should schedule a follow-up with the IDX/Dev-Ex team to discuss if `idx-js` is the best place to handle the redirect.

Is the SIW design review the best place for this, or are there alternate office hours for IDX that would be more beneficial?

Also, in regards to errors: we're going to need to rely on the `idx` response to correctly handle possible error states for us. These include:
- Errors that may occur within an OAuth 2.0 redirect (invalid config, token exp/iat skew, etc) which _should_ already be handled by the Widget's parsing of the [`error_description`](https://tools.ietf.org/html/rfc6749#section-5.2)
- User selects "sign-out" or there is a timeout in the factor enrollment/challenge.

Reply:
One more thing to explore is to add this redirect to `handleInvokeAction` instead of `handleSaveForm`.. Basically hitting on `Enroll` button for this flow we are actually not saving any form data?